web/.env.development

```diff
@@ -0,0 +1 @@
+ENV=development
```

Comment:
just curious is your prettier set up? I only ask because I don't see a new line here. And the auto-format workflow would not catch this because that only runs on app/

Reply:
No idea, i'll double check